### ISSUE TITLE
[Bug]: Media language is not configurable

### ISSUE THREAD

**Issue by Davidinho91** [sircharlo/meeting-media-manager#1004]
### What happened?

Bei den Einstellungen Medien: will ich Mediensprache noch einstellen aber es öffnet sich kein Dropdown. Keine Sprachen werden als Auswahl angezeigt, auch wenn ich zB Deutsch eingebe kommt nichts. 

### To Reproduce

3. Do '....'

### What did you expect to happen?

Das ich eine Spache auswählen kann wie bislang

### Version

v23.1.0

### What platform are you seeing the problem on?

Windows

### Relevant log output

```shell
{
  "1672746971942": [
    "socket hang up"
  ],
  "1672746971958": [
    "socket hang up"
  ],
  "1672746971960": [
    "socket hang up"
  ]
}
```

### Screenshots

![image](https://user-images.githubusercontent.com/121868614/210352933-5c962340-d08f-4d61-bf8d-596f54ef2412.png)

### Additional context

Windows_NT 10.0.22000 x64
### Anonymized `prefs.json`
```
{
  "app": {
    "autoOpenFolderWhenDone": false,
    "autoQuitWhenDone": false,
    "autoRunAtBoot": false,
    "autoStartSync": true,
    "congregationName": "Bergisch Gladbach - Homeoffice David",
    "disableAutoUpdate": true,
    "disableHardwareAcceleration": false,
    "localAppLang": "de",
    "localOutputPath": "***",
    "obs": {
      "enable": false,
      "port": null,
      "password": null,
      "mediaScene": null,
      "cameraScene": null,
      "useV4": true
    },
    "offline": false,
    "outputFolderDateFormat": "YYYY-MM-DD",
    "theme": "system"
  },
  "cong": {
    "server": null,
    "port": null,
    "user": null,
    "password": null,
    "dir": null
  },
  "media": {
    "enableMediaDisplayButton": false,
    "enableMp4Conversion": false,
    "enablePp": false,
    "enableSubtitles": false,
    "enableVlcPlaylistCreation": false,
    "excludeLffImages": false,
    "excludeTh": false,
    "hideMediaLogo": false,
    "hideWinAfterMedia": false,
    "includePrinted": false,
    "keepOriginalsAfterConversion": false,
    "lang": null,
    "langFallback": null,
    "langSubs": null,
    "langUpdatedLast": null,
    "maxRes": "720p",
    "mediaWinShortcut": "Alt+Z",
    "ppBackward": null,
    "ppForward": null,
    "preferredOutput": "window",
    "presentShortcut": "Alt+D"
  },
  "meeting": {
    "enableMusicButton": false,
    "enableMusicFadeOut": false,
    "musicFadeOutTime": 5,
    "musicFadeOutType": "smart",
    "musicVolume": 100,
    "mwDay": 2,
    "mwStartTime": "19:00",
    "shuffleShortcut": "Alt+K",
    "specialCong": false,
    "weDay": 6,
    "weStartTime": "13:00"
  },
  "__internal__": {
    "migrations": {
      "version": "23.1.0"
    }
  }
}
```

**Comment by mtdvlpr** [comment]
Hi @Davidinho91, could you check if the following file exists and has a list of languages: `C:/Users/%USERNAME%/AppData/Roaming/meeting-media-manager`

**Comment by sircharlo** [comment]
Any news about this? @Davidinho91

**Comment by sircharlo** [comment]
For what it's worth, I was able to simulate running the app using the provided preferences, without any issue. Perhaps the problem was in fact a corrupted or incorrectly formatted `langs.json` file. @mtdvlpr

**Comment by mtdvlpr** [comment]
@sircharlo, I fiess you're right, especially since `langUpdatedLast` is null! If it was corrupted, it should have been fixed at 770cc88, so I guess we'll close this issue.
